ksqldb-engine/src/main/java/io/confluent/ksql/planner/plan/DataSourceNode.java

```diff
@@ -104,7 +103,7 @@ public int getPartitions(final KafkaTopicClient kafkaTopicClient) {
   }
 
   @Override
-  public SchemaKStream<?> buildStream(final KsqlQueryBuilder builder) {
+  public SchemaKStream<?> buildStream(final PlanBuildContext builder) {
```

Comment:
nit: please don't hate me 😇 but there's quite a few places in this pr where we don't rename `builder` to something like `buildContext`. Probably makes sense to do it now while the diff is in front of us

Reply:
fair enough. I got lazy :)